### PR TITLE
Add BTRST Sepolia and Goerli and Base addresses

### PR DESCRIPTION
**Pull request by zencephalon**
**Description**

Adds addresses for Sepolia, Goerli, Base-Sepolia, and Base-Goerli for the BTRST token

**Tests**

No new tests.

### CODE REVIEW

**Comment by roninjin10** [comment]
lgtm does this need a base approval too?

**Comment by roninjin10** [comment]
Actually looks like a L1_STANDARD_BRIDGE_INFORMATION is missing. We will need to either add it now or if this can wait up to a week the new validation should be landing on monday or tuesday

File: data/BTRST/data.json [hunk 1]
```diff
@@ -14,6 +14,18 @@
...
+      "address": "0x39e09359F5a7396937504Eb2766722e2bf4650e8"
+    },
+    "base-goerli": {
+      "address": "0x99aC4484e8a1dbd6A185380B3A811913Ac884D87"
+    },
+    "sepolia": {
+      "address": "0xE490c7698F2b80789Fe312162d9D0a1Bd8C9beeE"
+    },
+    "base-sepolia": {
+      "address": "0xE490c7698F2b80789Fe312162d9D0a1Bd8C9beeE"
```
Comment (roninjin10) [inline data/BTRST/data.json:28]: Base sepolia is missing from chains.ts

Comment (zencephalon) [reply data/BTRST/data.json:28]: Yeah we need https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/634 merged first?

Comment (roninjin10) [reply data/BTRST/data.json:28]: Done can rebase this

Comment (zencephalon) [reply data/BTRST/data.json:28]: Thanks!

**Review by david-roth-cb — APPROVED** [review]

**Review by roninjin10 — APPROVED** [review]

**Pull request merged**